source/AI.cpp

```diff
@@ -2595,6 +2610,34 @@ void AI::DoScatter(Ship &ship, Command &command)
 
 
 
+// Elusive personality: cloaks whenever possible (even if it uses fuel)
+void AI::DoElusive(Ship &ship, Command &command)
+{
+	// Cloak if you can.
+	if(ship.Attributes().Get("cloak"))
+		command |= Command::CLOAK;
+	
+	// Cloak if your parent is cloaked.
+	const shared_ptr<const Ship> &parent = ship.GetParent();
+	if(parent && parent->Commands().Has(Command::CLOAK))
+		command |= Command::CLOAK;
```

Comment:
The second part of this adds no value -- if the ship is capable of cloaking, then it will already have set the cloak command. If it isn't capable of cloaking, then it won't have the command set, and setting it because the parent is attempting to cloak is meaningless.

Reply:
gotcha